addons/hr_holidays/tests/common.py

```diff
@@ -16,6 +16,18 @@ def setUpClass(cls):
         cls.company = cls.env['res.company'].create({'name': 'Test company'})
         cls.env.user.company_id = cls.company
 
+        # The available time off type are the ones whose:
+        # 1. Company is one of the selected companies.
+        # 2. Company is false but whose country is one the countries of the selected companies.
+        # 3. Company is false and country is false
+        # Thus, a time off type is defined to be available for `Test company`
+        cls.env['hr.leave.type'].create({
+            'name': 'Test Leave Type',
+            'requires_allocation': 'no',
+            'request_unit': 'day',
+            'company_id': cls.company.id,
+        })
```

Comment:
Is this used somewhere? It might also be nice to have a test ensuring that the security constraint is working as expected

Reply:
Good to know! It might be nice to add a comment so that we know why it's there